mlflow/lightgbm.py

```diff
@@ -298,11 +298,19 @@ def autolog(log_input_examples=False, log_model_signatures=True):
     :param log_input_examples: If ``True``, input examples from training datasets are collected and
                                logged along with LightGBM model artifacts during training. If
                                ``False``, input examples are not logged.
+                               Note: if ``log_models`` is set to ``False``,
+                               then the value of ``log_input_examples``
+                               will be ignored and not logged.
     :param log_model_signatures: If ``True``,
                                  :py:class:`ModelSignatures <mlflow.models.ModelSignature>`
                                  describing model inputs and outputs are collected and logged along
                                  with LightGBM model artifacts during training. If ``False``,
                                  signatures are not logged.
+                                 Note: if ``log_models`` is set to ``False``,
+                                 then the value of ``log_model_signatures``
+                                 will be ignored and not logged.
+    :param log_models: If ``True``, the trained model will be logged as an artifact. If ``False``,
```

Comment:
For consistency with the documentation of `log_input_examples` and `log_model_signatures`, which are written to reflect that multiple models may be logged over the course of an autologging workflow, can we adjust the phrasing here as follows?:

```
If ``True``, trained models are logged as MLflow model artifacts. If ``False``, trained models are not logged. Input examples and model signatures, which are attributes of MLflow models, are also omitted when ``log_models`` is ``False``.
```

Can we make this adjustment for all relevant autologging methods that we're documenting?